apps/src/templates/progress/LessonGroup.jsx

```diff
@@ -121,7 +124,10 @@ class LessonGroup extends React.Component {
 
     const hasVisibleLesson = lessons.some(lesson => lessonIsVisible(lesson));
 
-    if (!hasVisibleLesson) {
+    if (
```

Comment:
Would this be equivalent to `!hasVisibleLesson && viewAs === ViewType.Student` (which seems to more directly express the condition)?  Or possibly `return null unless hasVisibleLesson || viewAs === ViewType.Teacher`?

Reply:
Yes - I think you're right, that is the equivalent, I'll re-write. `unless` is just for ruby right?

Reply:
Yes, my second suggestion is probably the more "ruby-ish" way to write the condition but some people find it confusing.  Personally, I'm a bit on the fence about `unless` because it sometimes leads to confusing double-negatives, but here I think it actually reads pretty clearly.  Up to you.

Reply:
Oh oops, for some reason, I got confused and thought we were on the back-end.  Disregard all of the `unless` nonsense.  Sorry!

Reply:
No problem!